### PR TITLE
Fix more typos

### PR DESCRIPTION
**Pull request by ItayZiv**
Fix Typos 2: Electric Boogaloo
Akin to https://github.com/wpilibsuite/frc-docs/pull/673 .
Includes a few more logic-based fixes (code/consistancy etc...)
Any more typos found are welcome to be commented and ill throw them in

Stuff to be done:

- [x] CommandScheduler link at https://docs.wpilib.org/en/latest/docs/software/commandbased/what-is-command-based.html#how-commands-are-run
- [x] Make comments consistant https://docs.wpilib.org/en/latest/docs/software/commandbased/subsystems.html#creating-a-subsystem

### CODE REVIEW

File: source/docs/software/commandbased/what-is-command-based.rst [hunk 1]
```diff
@@ -27,20 +27,21 @@ In contrast, in an ordinary `imperative <https://en.wikipedia.org/wiki/Imperativ
...
   .. code-tab:: c++
 
     if(aButton.Get()) {
       if(!pressed) {
         Intake.Run();
         pressed = true;
-      } else {
-        pressed = false;
-      }
+      }
```
Comment (ItayZiv) [inline source/docs/software/commandbased/what-is-command-based.rst:42]: Remove trailing whitespace

**Comment by sciencewhiz** [comment]
PRs are cheap. I don't recommend holding a typo PR in draft so it can't be merged. If more issues are found, a new PR can be opened.

**Comment by ItayZiv** [comment]
Yea I had some more stuff to fix, thought I already marked as ready for review today though

File: source/docs/software/commandbased/what-is-command-based.rst [hunk 2]
```diff
@@ -27,20 +27,21 @@ In contrast, in an ordinary `imperative <https://en.wikipedia.org/wiki/Imperativ
       if(!pressed) {
         intake.run();
         pressed = true;
-      } else {
-        pressed = false;
       }
     }
+    else {
```
Comment (AustinShalit) [inline source/docs/software/commandbased/what-is-command-based.rst:32]: This should move up a line to match the style guide.

**Pull request merged**